### ISSUE TITLE
App threw an error during load

### ISSUE THREAD

**Issue by c-p-dev** [leota/electron-angular4-sqlite3#17]
Hi,

I was looking for similar setup and come across your project. I hope you don't mind. 

I am encountering an error running "npm run electron-win ".  I not quite sure why knex throwing out error. 

App threw an error during load
D:\electron-sample\electron-angular4-sqlite3-master\node_modules\knex\src\migrate\Migrator.js:43
          ...knex.userParams,
          ^^^
SyntaxError: Unexpected token ...
    at createScript (vm.js:53:10)
    at Object.runInThisContext (vm.js:95:10)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (D:\electron-sample\electron-angular4-sqlite3-master\node_modules\knex\src\util\make-knex.js:3:22)
    at Object.<anonymous> (D:\electron-sample\electron-angular4-sqlite3-master\node_modules\knex\src\util\make-knex.js:284:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)

Thanks for the help.

**Comment by leota** [comment]
Can you please provide some more details about OS, Node.js version, NPM version?

**Comment by c-p-dev** [comment]
I am using
Node.js v10.14.1
NPM 6.4.1
OS: Windows 10 Pro 64-bit

I'll try to run this project with updated versions on my other computer. Thanks!

**Comment by leota** [comment]
Just tested on a Windows 10 Pro 64-bit machine. Everything runs smoothly.
Here's my setup:

- Node.js v10.13.0
- npm 6.4.1

**Globally installed npm packages:**

-  node-gyp@3.6.2
-  windows-build-tools@1.3.2

Hope it helps. Let me know!

**Comment by c-p-dev** [comment]
Hmm. I've tried to run it with my other computer and I have updated all my packages.

I think it comes down to my angular/cli version. my global cli version is 7.1.2. 
Im trying to use angular/cli version of this current project but for some reason it still fails. 

I think I need to try changing my angular/cli version to 4 but I really dont want to downgrade my global angularcli version.

Thanks btw for the help!
